applications/callflow/doc/cf_branch_variable.md

```diff
@@ -0,0 +1,70 @@
+### Callflow Branch on a Variable's Value
+
+#### Overview
+
+The `branch_variable` callflow enables you to branch based on value of some field inside one of the a call CCVs, user's document, device's documten or an account's document.
+
+##### Callflow fields
+
+Key | Description | Type | Default | Required
+--- | ----------- | ---- | ------- | --------
+`variable` | Name of variable/property that we are looking for | `string` or a json path | | `true`
+`scope` | the place which the variable should be looked for (`custome_channel_vars`, `device`, `user`, `account`) | `string` | `"cunstom_channel_vars"` | `false`
```

Comment:
sorry, I think the documentation is a little bit confusing, if scope is defined but it's value is not `device`, `user` or `account` (e.g. it sets to `merged`) it will operate on kz_endpoint.